Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -46,6 +46,11 @@ struct TestCommand: ParsableCommand {
     )
     var configuration: String?
 
+    @Flag(
+        help: "When passed, it skips testing UI Tests targets."
+    )
+    var skipUiTests: Bool = false
```

Comment:
Should we follow SPM here and use flag inversion, so: `--no-ui-tests` [here](https://github.com/apple/swift-argument-parser/blob/main/Documentation/02%20Arguments%2C%20Options%2C%20and%20Flags.md#using-flag-inversions-enumerations-and-counts). So we'd have `--ui-tests` enabled by default and then `--no-ui-tests` if users wish to not use it.

Reply:
Similar for `--no-unit-tests` with `--unit-tests` enabled by default

Reply:
Can you please link the example of SPM command that uses this convention? By printing a help via `swift package --help` there's the flags like 

```
--skip-update           Skip updating dependencies from their remote during a resolution
```

IMHO `--skip-ui-tests` is easy to understand (no negation as it was with `no-ui-tests`) and it expresses intent clearly. 

Let see what others think. 

IMHO `--skip-ui-tests` flag is easy to read (no negation as it was with `--no-ui-tests`)

Reply:
I'm personally inclined towards `--skip-ui-tests` because `skip` captures the intent of the flag, where as `no` is a bit ambiguous.